CHANGELOG.md

```diff
@@ -26,6 +26,60 @@ Packages are formatted by release version. You can see all the changes done to p
+
+### :bug: Bug Fixes
+* [project-base] returned symfony.lock to project-base as it caused problems with symfony flex installing packages again by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2595
+* [framework] AdministrationFacade: support requests without _route parameter by @mposchl in https://github.com/shopsys/shopsys/pull/2373
+* [shopsys] updated acceptance tests because they were no longer working locally by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2610
+* [shopsys] too many attempts js error is now translated on frontend by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2613
+* [framework] removed class duplication in expanded choice by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2614
+
+### :hammer: Developer experience and refactoring
+* [shopsys] removed duplicated CreateReleaseNotesReleaseWorker by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2579
+* [framework] added missing getters to UploadedFile by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2582
+* [framework] bin/console base phing targets are now more verbose by @vitek-rostislav in https://github.com/shopsys/shopsys/pull/2507
+* [framework] Transport::getPrice() - fix error message by @vitek-rostislav in https://github.com/shopsys/shopsys/pull/2506
+* [framework] added commonly used methods to Price class by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2589
+* [framework] build-demo-dev-quick phing target now correctly recreates elasticsearch structure by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2590
+* [framework] removed DRY violation in AggregationResultToProductFilterCountDtaTransformer and fixed property name by @sspooky13 in https://github.com/shopsys/shopsys/pull/2501
+* [shopsys] checking if order status update email should be sent is done directly in OrderMailFacade instead in multiple places by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2588
+* [framework] fixed upgrade notes and resolved possible BC-break by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2598
+* [shopsys] updated installation docs for macOS using Mutagen by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2593
+* [shopsys] added BlankLineBeforeStatementFixer for ensuring empty line before statements by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2128
+* [framework] RegisterProjectFrameworkClassExtensionCompilerPass now ensures that both class and interface alias are registered in Symfony services by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2605
+* [project-base] config folder is now less confusing by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2607
+
+### :link: Frontend API
+* authentication failure now return errors as array by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2387
+* empty strings are now considered nulls in frontend API by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2601
```

Comment:
These two are missing bracketed package that it belongs to - do we want to add it or remove it everywhere?

Reply:
added in c110a6e27